### PR TITLE
chore: inbox react docs improvements

### PR DESCRIPTION
**Pull request by LetItRock**
Improvements to the Inbox React documentation.

https://github.com/user-attachments/assets/b5a631cd-962e-4c72-b4c8-e43151149f92

### CODE REVIEW

File: inbox/introduction.mdx [hunk 1]
```diff
@@ -1,44 +1,33 @@
...
 
 In-App notifications are different from push notifications in the sense that Inbox notifications are consumed by the user in your app itself, whereas push notifications are shown on the device level. Read more about [push channel and supported providers.](/channels-and-providers/push/overview)
 
-Novu provides a pre-built, ready-to-use, and customizable UI component. It also allows you to bring your own UI via a headless component which can be used to add In-App notifications to any platform.
+Novu provides a pre-built, ready-to-use, and customizable Inbox UI component.
 
 ## Example
 
-[![Edit kind-nova-xv6s9g](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/kind-nova-xv6s9g?fontsize=14&hidenavigation=1&theme=dark)
+[![Edit kind-nova-xv6s9g](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/devbox/pensive-ritchie-cjwvc5)
```
Comment (LetItRock) [inline inbox/introduction.mdx:14]: I created the new codesandbox with new Inbox React component

File: inbox/introduction.mdx [hunk 2]
```diff
@@ -1,44 +1,33 @@
...
-5. `redirect url`:- When a single notification item from the list is clicked then it can be configured to perform an action or redirect to any URL.
-
-## UI Libraries
-
-1. [React](/notification-center/client/react/get-started)
-2. [Vue](/notification-center/client/vue)
-3. [Angular](/notification-center/client/angular)
-4. [Web Component](/notification-center/client/web-component)
-5. [Headless](/notification-center/client/headless/get-started)
-
```
Comment (LetItRock) [inline inbox/introduction.mdx:41]: Removed as we only support React as of now

File: inbox/introduction.mdx [hunk 3]
```diff
@@ -1,44 +1,33 @@
...
-## Backend SDK Methods
-
-[Backend sdk methods](/notification-center/backend/methods)
```
Comment (LetItRock) [inline inbox/introduction.mdx:44]: Moved to Advanced Configuration page

File: inbox/react/components.mdx [hunk 4]
```diff
@@ -19,53 +19,50 @@ It renders an opinionated `<Inbox />` with a bell button, that triggers a popove
...
 
 function Novu() {
   return (
     <Inbox
-      options={{
-        subscriberId: "SUBSCRIBER_ID",
-        applicationIdentifier: "APPLICATION_IDENTIFIER",
-      }}
+      applicationIdentifier="YOUR_APPLICATION_IDENTIFIER"
+      subscriberId="YOUR_SUBSCRIBER_ID"
```
Comment (LetItRock) [inline inbox/react/components.mdx:29]: Here we have mainly improvements on the props, however some sections were duplicated with a separate page.

File: inbox/react/components.mdx [hunk 5]
```diff
@@ -103,45 +98,19 @@ function Novu() {
 }
 ```
 
-#### Multiple Tabs
```
Comment (LetItRock) [inline inbox/react/components.mdx:106]: We have a dedicated page

File: inbox/react/customization.mdx [hunk 6]
```diff
@@ -1,205 +1,45 @@
...
 - `baseTheme`: The baseTheme object allows you to define the base theme for the inbox. This can include variables and elements that apply to the entire inbox, as well as additional themes that can be switched between. It accepts an array of objects, each containing variables and elements. Each object will override the previous object's styles.
   This enables to override the custom themes imported from the `@novu/react` package.
-  Currently, we only export the `dark` theme.
+  Currently, we only export the `dark` [theme](https://github.com/novuhq/novu/blob/01268c688d1c173736b4ad51147370844952335a/packages/js/src/ui/config/defaultVariables.ts#L3).
 
-```jsx
+```tsx
 import { Inbox } from '@novu/react';
 const appearance = {
-          variables: {
```
Comment (LetItRock) [inline inbox/react/customization.mdx:21]: This code might become quickly outdated, so I provided a small amount to show what is possible. The appearance prop section has the link to the code where they can check all possible properties.

File: inbox/react/get-started.mdx [hunk 7]
```diff
@@ -1,152 +1,43 @@
...
-      options={{
-        subscriberId: "SUBSCRIBER_ID",
-        applicationIdentifier: "APPLICATION_IDENTIFIER",
-      }}
-    />
-  );
-}
-```
-
-## Use your own backend and socket URL
```
Comment (LetItRock) [inline inbox/react/get-started.mdx:32]: Moved to Advanced Config page

File: inbox/react/preference.mdx [hunk 8]
```diff
@@ -1,54 +1,13 @@
...
 
 Users can enable/disable any active channel in the workflow using subscriber preferences or they can update the preference globally for all workflows under the `Global Preferences`.
 
 <img
-  style={{ borderRadius: "0.5rem" }}
+  style={{ borderRadius: '0.5rem' }}
   src="/images/notification-center/client/react/preferences/preferences.png"
 />
-
-## Get subscriber preferences
```
Comment (LetItRock) [inline inbox/react/preference.mdx:15]: Not relevant here

File: mint.json [hunk 9]
```diff
@@ -125,21 +125,8 @@
...
-      "icon": "code",
-      "pages": ["inbox/headless/get-started", "inbox/headless/api-reference"]
-    },
-
-    {
-      "group": "Other Frameworks",
-      "pages": [
-        "inbox/client/vue",
-        "inbox/client/angular",
-        "inbox/client/web-component"
```
Comment (LetItRock) [inline mint.json:142]: not needed

File: inbox/introduction.mdx (hunk 1 above)
Comment (rifont) [reply inbox/introduction.mdx:14]: Wdyt about using Stackblitz instead? It boots much faster

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**